### ISSUE TITLE
Feature: Allow app to receive `KeyEvent` from Accessibility Service

### ISSUE THREAD

**Issue by fahimfoysal2** [X-SLAYER/flutter_accessibility_service#19]
**_Feature Request_**
- As Accessibility Service also provides key press events listener, can we  please have this feature? 
- Do you have any plan for it?
- If not, can you give me a guideline how can I implement it? 

Thanks

**Comment by X-SLAYER** [comment]
The package already have this feature  `FlutterAccessibilityService.performAction` check the example folder and you will find some code examples

**Comment by fahimfoysal2** [comment]
> The package already have this feature `FlutterAccessibilityService.performAction` check the example folder and you will find some code examples

Thanks for your quick reply. 

Sorry, maybe I was not clear. 
By receiving key press events, I meant to receive callbacks via the following method 

```java
protected boolean onKeyEvent(KeyEvent event) { throw new RuntimeException("Stub!"); }
```
from AccessibilityService class
for keys like **_volume up/volume down buttons_**.

---
I started the implementation by overriding the method in AccessibilityListener.java

```java
@Override
protected boolean onKeyEvent(KeyEvent event) {
        HashMap<String, Object> eventData = new HashMap<>();
        eventData.put("keyCode", event.getKeyCode());
        eventData.put("action", event.getAction());
        // Add more properties as needed

        // Send the event data to Flutter

        return super.onKeyEvent(event);
}
```

**Comment by X-SLAYER** [comment]
Yes for the moment this feature is not implemented yet. maybe in the next versions or else any pull requests are welcomed

**Comment by fahimfoysal2** [comment]
> Yes for the moment this feature is not implemented yet. maybe in the next versions or else any pull requests are welcomed

If you can guide me, or suggest a flow for the implementation, I can try.
